src/auto-evo/AutoEvoExploringTool.tscn

```diff
@@ -0,0 +1,763 @@
+margin_right = 330.0
+margin_bottom = 20.0
+custom_fonts/font = ExtResource( 4 )
+text = "SPECIES_DETAILS"
+
+[node name="MarginContainer" type="MarginContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer"]
+margin_top = 24.0
+margin_right = 330.0
+margin_bottom = 486.0
+size_flags_horizontal = 3
+size_flags_vertical = 3
+custom_constants/margin_right = 10
+custom_constants/margin_top = 10
+custom_constants/margin_left = 10
+custom_constants/margin_bottom = 10
+
+[node name="ScrollContainer" type="ScrollContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer/MarginContainer"]
+margin_left = 10.0
+margin_top = 10.0
+margin_right = 320.0
+margin_bottom = 452.0
+
+[node name="VBoxContainer" type="VBoxContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer/MarginContainer/ScrollContainer"]
+margin_right = 310.0
+margin_bottom = 442.0
+size_flags_horizontal = 3
+size_flags_vertical = 3
+
+[node name="HBoxContainer" type="HBoxContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer/MarginContainer/ScrollContainer/VBoxContainer"]
+margin_right = 310.0
+margin_bottom = 200.0
+rect_min_size = Vector2( 0, 200 )
+
+[node name="CellHexPreview" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer/MarginContainer/ScrollContainer/VBoxContainer/HBoxContainer" instance=ExtResource( 10 )]
+margin_right = 153.0
+margin_bottom = 200.0
+
+[node name="SpeciesPreview" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer/MarginContainer/ScrollContainer/VBoxContainer/HBoxContainer" instance=ExtResource( 9 )]
+margin_left = 157.0
+margin_right = 310.0
+margin_bottom = 200.0
+
+[node name="SpeciesInfoLabel" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer/MarginContainer/ScrollContainer/VBoxContainer" instance=ExtResource( 5 )]
+margin_top = 204.0
+margin_right = 310.0
+margin_bottom = 442.0
+size_flags_horizontal = 3
+size_flags_vertical = 3
+scroll_active = false
+
+[node name="HBoxContainer" type="HBoxContainer" parent="GUI/VBoxContainer"]
+margin_top = 625.0
+margin_right = 1180.0
+margin_bottom = 660.0
+
+[node name="BackButton" type="Button" parent="GUI/VBoxContainer/HBoxContainer"]
+margin_right = 62.0
+margin_bottom = 35.0
+text = "BACK"
+
+[node name="ExitConfirmationDialog" parent="." instance=ExtResource( 11 )]
+rect_min_size = Vector2( 400, 0 )
+WindowTitle = "CONFIRM_EXIT"
+DialogText = "RETURN_TO_MENU_WARNING"
+
+[connection signal="pressed" from="GUI/VBoxContainer/TabButtons/ConfigButton" to="." method="ChangeTab" binds= [ 0 ]]
```

Comment:
Elsewhere we use the approach where the selected tab is an enum and the binds is a valid enum value like "Structure" for the editor structure tab. I think the same pattern should be used here for codebase consistency reasons.